test/support/sdk/TestEnvironment.ts

```diff
@@ -221,13 +220,14 @@ export class TestEnvironment {
     }
 
     let html = '<!doctype html><html><head></head><body></body></html>';
+    global.document = jsdom.jsdom('');
```

Comment:
Why was this line required?

Reply:
the global document object must be defined for `getSlidedownElement` to work correctly (below). this line initializes the document object so that function works.

Reply:
ah, gotcha 👍